.gitlab/benchmarks.yml

```diff
@@ -51,6 +51,11 @@ benchmark-span:
   variables:
     SCENARIO: "span"
 
+benchmark-otel-tracer:
+  extends: .benchmarks
+  variables:
+    SCENARIO: "otel_tracer"
+
```

Comment:
Failing against 1.x, I may need to introduce the gitlab benchmarks in a follow up PR

```suggestion
```